src/app/app-routing.module.ts

```diff
@@ -1,7 +1,42 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { LoginComponent } from './login/login.component';
+import { LayoutComponent } from './layout/layout.component';
+import { HomeComponent } from './home/home.component';
+import { NotesComponent } from './notes/notes.component';
+import { PageNotFoundComponent } from './page-not-found/page-not-found.component';
+import { GuardService } from './services/guard.service';
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: 'login', component: LoginComponent },
+  { path: '',   redirectTo: 'login', pathMatch: 'full' },
+  { 
+    path: 'app', 
+    component: LayoutComponent,
+    children: [
+      {
+        path: 'home',
+        component: HomeComponent,
+      },
+      {
+        path: 'notes',
+        component: NotesComponent,
+      },
+      { 
+        path: '',   
+        redirectTo: 'home', 
+        pathMatch: 'full' 
+      },
+      { 
+        path: '**', 
+        component: PageNotFoundComponent 
+      },
+    ],
+    canActivate: [GuardService]
```

Comment:
What happens if you try to go to a url like /app/notes? Does this guard still work?